### PR TITLE
Add support for circe (close #37)

### LINKED ISSUE #37: Add circe support
[Circe] (https://github.com/PDXostc/rvi_sota_server/tree/master/common/src/main/scala/org/genivi/sota/marshalling) now seems to be very interesting and actively developing JSON library.
I've seen at least [one implementation](https://github.com/PDXostc/rvi_sota_server/tree/master/common/src/main/scala/org/genivi/sota/marshalling) can be used here if @koshelev wouldn't mind.

### PR DESCRIPTION
**Pull request by chuwy**

### CODE REVIEW

File: akka-http-circe/src/main/scala/de/heikoseeberger/akkahttpcirce/CirceSupport.scala [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+package de.heikoseeberger.akkahttpcirce
+
+import akka.http.scaladsl.marshalling.{ Marshaller, ToEntityMarshaller }
+import akka.http.scaladsl.model.{ ContentTypes, HttpCharsets, MediaTypes }
+import akka.http.scaladsl.unmarshalling.{ FromEntityUnmarshaller, Unmarshaller }
+
+import cats.data.Xor
+
+import io.circe._
```
Comment (hseeberger) [inline akka-http-circe/src/main/scala/de/heikoseeberger/akkahttpcirce/CirceSupport.scala]: No wildcard imports at top-level, please. Also no new lines btw import statements.

**Comment by hseeberger** [comment]
Looks fantastic! Just one minor style issue.

**Comment by chuwy** [comment]
Fixed.

**Comment by hseeberger** [comment]
Thanks!

**Comment by hseeberger** [comment]
Published version 1.3.0-3-gf43c38f to `hseeberger at bintray" at "http://dl.bintray.com/hseeberger/maven"`.

**Comment by jpfuentes2** [comment]
Awesome, thanks @chuwy :)

**Pull request merged**